docs/_snippets/features/keyboard-support.js

```diff
@@ -215,8 +215,8 @@ ClassicEditor
 		exportPdf: {
 			stylesheets: [
 				'../../assets/pagination-fonts.css',
-				'EDITOR_STYLES',
-				'../../snippets/features/pagination/snippet.css',
+				'../../assets/ckeditor5/ckeditor5.css',
```

Comment:
I wouldn't say I like the relative paths, but I don't want to spend more on it than needed, so let's leave it and improve in the future (if it becomes a problematic case).